validate.py

```diff
@@ -137,11 +131,20 @@ def generate_result_text(query_name, checked_query_output_df):
     job_name = job["full_name"]
     query_keys = job["queries"]
 
+    # Set up connections to data sources used by job's queries
+    db_conns = {}
+    data_sources_for_queries = [QUERIES[query_key]['data_source'] for query_key in query_keys]
+    data_sources_used_by_job = pd.Series(data_sources_for_queries).drop_duplicates().to_list()
```

Comment:
Using Pandas just to drop duplicates seems unnecessary here. . . 

Since we don't care about order maybe just use a set instead of a list in the first place. 
https://www.geeksforgeeks.org/python-ways-to-remove-duplicates-from-list/

Reply:
Hm..., yeah, using a list comprehension is probably simpler here. I'm already doing this for the flags, so that would need to be changed for consistency.

Reply:
We're already using Pandas anyway so I'd probably just consider this minor.